minemeld/ft/phishme.py

```diff
@@ -71,6 +72,7 @@ def configure(self):
         })
         self.product = self.config.get('product', 'malware')
         self.source_name = self.config.get('source_name', 'phishme.intelligence')
+        self.user_agent = {'user-agent': _API_USER_AGENT}
```

Comment:
I would rename the self.*user_agent* self.*headers* as this contains the full dictionary of headers for the request.